Source/JavaScriptCore/inspector/protocol/DOM.json

```diff
@@ -239,7 +240,8 @@
                 { "name": "height", "type": "integer", "description": "The native height of the video track in CSS pixels" },
                 { "name": "width", "type": "integer", "description": "The native width of the video track in CSS pixels" },
                 { "name": "spatialVideoMetadata", "$ref": "SpatialVideoMetadata", "optional": true },
-                { "name": "videoProjectionMetadata", "$ref": "VideoProjectionMetadata", "optional": true }
+                { "name": "videoProjectionMetadata", "$ref": "VideoProjectionMetadata", "optional": true },
+                { "name": "isProtected", "type": "boolean", "description": "Whether the track contains protected contents" }
```

Comment:
ditto

if for whatever reason you dont want to make this optional then please move this above the other optional properties as we try to keep optional properties last for better readability at a glance